src/main/kotlin/org/rust/ide/inspections/fixes/AddMainFnFix.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsFile
+import org.rust.lang.core.psi.RsPsiFactory
+
+class AddMainFnFix(file: RsFile) : LocalQuickFixAndIntentionActionOnPsiElement(file) {
+
+    override fun getFamilyName(): String = "Add `main` fn"
+
+    override fun getText(): String = "Add `fn main()`"
```

Comment:
I don't think it makes sense to have different text in `getFamilyName` and `getText`